### ISSUE TITLE
Add `ConfigurationSchema` tool to repo if possible

### ISSUE THREAD

**Issue by Alirexaa** [CommunityToolkit/Aspire#112]
I'll check with the engineering team but I think I was told that the tool is an internal MS tool.

_Originally posted by @aaronpowell in https://github.com/CommunityToolkit/Aspire/pull/66#discussion_r1791660943_

**Comment by bart-vmware** [comment]
There's an open issue at https://github.com/dotnet/aspire/issues/3309.

**Comment by Alirexaa** [comment]
@aaronpowell, I like the `Stale` tag to be added for issues that already have `awaiting response`. We have multiple issues that cannot resolved at this time but we want to fix them at some point like this issue,

**Comment by aaronpowell** [comment]
yeah we should add a way to have it just ignore stale, have to tweak the workflow.

The last tweak was to ensure we didn't have requests for integrations that are never actioned (as there's no one who has the time/knowledge to build them).
